daemon/algod/api/server/v2/handlers.go

```diff
@@ -918,9 +918,29 @@ func (v2 *Handlers) RawTransaction(ctx echo.Context) error {
 	return ctx.JSON(http.StatusOK, model.PostTransactionsResponse{TxId: txid.String()})
 }
 
+// preEncodedTxnResult mirrors simulation.TxnResult
+type preEncodedTxnResult struct {
+	Txn              PreEncodedTxInfo `codec:"txn-result"`
+	MissingSignature *bool            `codec:"missing-signature,omitempty"`
+}
+
+// preEncodedTxnGroupResult mirrors simulation.TxnGroupResult
+type preEncodedTxnGroupResult struct {
+	Txns           []preEncodedTxnResult `codec:"txn-results"`
+	FailureMessage *string               `codec:"failure-message,omitempty"`
+	FailedAt       *[]uint64             `codec:"failed-at,omitempty"`
+}
+
+// preEncodedSimulationResult mirrors simulation.Result
+type preEncodedSimulationResult struct {
+	Version      uint64                     `codec:"version"`
+	TxnGroups    []preEncodedTxnGroupResult `codec:"txn-groups"`
+	WouldSucceed bool                       `codec:"would-succeed"`
+}
```

Comment:
We create these custom response types for the same reason we need `PreEncodedTxInfo`, which is explained in #1015

Reply:
also no missing signature?

Reply:
`MissingSignature` is present for each txn